src/TConsole.cpp

```diff
@@ -1724,10 +1724,13 @@ void TConsole::setHorizontalScrollBar(bool isEnabled)
     }
 }
 
-void TConsole::setScrolling(bool enabled)
+void TConsole::setScrolling(const bool state)
 {
     if (mType & (UserWindow | SubConsole)) {
-        mScrollingEnabled = enabled;
+        mScrollingEnabled = state;
+        if (!state) {
```

Comment:
Maybe this would be more intuitive:

```suggestion
        if (!mScrollingEnabled) {
```